python/cudf_polars/cudf_polars/dsl/expressions/boolean.py

```diff
@@ -31,22 +30,49 @@
 __all__ = ["BooleanFunction"]
 
 
+class BooleanFunctionName(Enum):
+    All = auto()
+    AllHorizontal = auto()
+    Any = auto()
+    AnyHorizontal = auto()
+    IsBetween = auto()
+    IsDuplicated = auto()
+    IsFinite = auto()
+    IsFirstDistinct = auto()
+    IsIn = auto()
+    IsInfinite = auto()
+    IsLastDistinct = auto()
+    IsNan = auto()
+    IsNotNan = auto()
+    IsNotNull = auto()
+    IsNull = auto()
+    IsUnique = auto()
+    Not = auto()
+
+    @staticmethod
+    def get_polars_type(tp: BooleanFunctionName):
+        function, name = str(tp).split(".")
+        if function != "BooleanFunction":
+            raise ValueError("BooleanFunction required")
+        return getattr(BooleanFunctionName, name)
```

Comment:
I think this doesn't return a polars object, but one of ours. Also, because we're calling getattr, that suggests we should make this a classmethod

```
@classmethod
def from_polars(cls, obj: pl_expr.BooleanFunction) -> Self:
    function, name = str(obj).split(".", maxsplit=1)
    if function != "BooleanFunction":
        raise ValueError("BooleanFunction required")
    return getattr(cls, name)
```